packages/story-editor/src/app/media/utils/test/useFFmpeg.js

```diff
@@ -44,25 +45,7 @@ function arrange({ userSettingEnabled, hasUploadMediaAction }) {
     },
   };
   const configState = {
-    allowedTranscodableMimeTypes: [
-      'video/3gpp',
-      'video/3gpp2',
-      'video/MP2T',
-      'video/mp4',
-      'video/mpeg',
-      'video/ogg',
-      'video/quicktime',
-      'video/webm',
-      'video/x-flv',
-      'video/x-h261',
-      'video/x-h263',
-      'video/x-m4v',
-      'video/x-matroska',
-      'video/x-mjpeg',
-      'video/x-ms-asf',
-      'video/x-msvideo',
-      'video/x-nut',
-    ],
+    allowedTranscodableMimeTypes: TRANSCODABLE_MIME_TYPES,
```

Comment:
This is unneeded. With `TRANSCODABLE_MIME_TYPES` being a constant in the editor now it never needs to be part of the config context. So this line can be removed.